asv_bench/benchmarks/series_methods.py

```diff
@@ -31,6 +32,14 @@ def time_constructor_fastpath(self):
         Series(self.array, index=self.idx2, name="name", fastpath=True)
 
 
+class SeriesConstructorEa:
+    def setup(self):
+        self.data = np.array(list(range(1_000_000)))
+
+    def time_constructor(self):
+        Series(data=self.data, dtype="Int64")
```

Comment:
I think we already have a benchmark for this in array.py (IntegerArray::time_from_integer_array), except that that one is with a tiny array and thus won't cover this aspect. But maybe add a version with a larger array in the existing benchmark? (or just make the array in that benchmark bigger)

Reply:
Thx, forgot to check for series after I found nothing for value_counts. Increased the array size for the existing benchmark. 4 values is probably a bit small to see anything with overhead from other calls that don't depend on array size